dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java

```diff
@@ -43,6 +43,7 @@
     private final AtomicLong _startTime = new AtomicLong();
     private final ApplicationMonitor _appMonitor;
     private final boolean _update;
+    private boolean cancelled = false;
```

Comment:
Should this be volatile? I'm pretty sure it should be, it looks like it would be read from one thread and written with another.